src/docker.cc

```diff
@@ -46,6 +46,17 @@ constexpr const char kDockerContainerResourcePrefix[] = "container";
 DockerReader::DockerReader(const MetadataAgentConfiguration& config)
     : config_(config), environment_(config) {}
 
+const bool DockerReader::IsConfigured() const {
+  try {
+    QueryDocker(std::string(kDockerEndpointPath) + "/json?all=true");
```

Comment:
It would be useful to minimize output by using limit / filter options:
https://docs.docker.com/engine/api/v1.24/#31-containers

Reply:
Updated.